nightly/expensive.txt

```diff
@@ -28,8 +28,8 @@ expensive --timeout=1800 near-client near_client tests::catching_up::test_catchu
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving --features nightly_protocol,nightly_protocol_features
 
-expensive integration-tests test_catchup test_catchup
-expensive integration-tests test_catchup test_catchup --features nightly_protocol,nightly_protocol_features
+expensive integration-tests test_catchup tests::test_catchup::test_catchup
```

Comment:
I’m pretty sure you need to change the third value from `test_catchup` to `integration_tests`. Similarly throughout the whole file.

Reply:
@mina86 Ok, I fixed it.